frontend/packages/llama-stack-modular-ui/bff/internal/api/openapi_handler.go

```diff
@@ -59,9 +62,34 @@ func NewOpenAPIHandler(logger *slog.Logger) (*OpenAPIHandler, error) {
 		logger:   logger,
 		spec:     spec,
 		specYAML: specData,
+		config:   cfg,
 	}, nil
 }
 
+// getDynamicSpec returns the OpenAPI spec with dynamic path prefixes
+func (h *OpenAPIHandler) getDynamicSpec() *openapi3.T {
+	// Create a deep copy of the spec by marshaling and unmarshaling JSON
+	specJSON, _ := json.Marshal(h.spec)
+
+	// Replace generic /api/v1 with the configured API path prefix in JSON
+	jsonContent := string(specJSON)
+	jsonContent = strings.ReplaceAll(jsonContent, "/api/v1", h.config.APIPathPrefix)
+
+	// Unmarshal back to spec
+	dynamicSpec := &openapi3.T{}
+	json.Unmarshal([]byte(jsonContent), dynamicSpec)
```

Comment:
Looks like the PR check is failing in this PR due to this line. This should log an error if any.

Reply:
thanks, yep, fixed it